README.md

```diff
@@ -160,75 +163,57 @@ You should use namespaces and Composer for autoloading.
 
 For projects that can't/won't rely on a transpiler, the recommended module syntax is [CommonJS](http://www.commonjs.org/). This is the same syntax used by Node.js, and is more similar to ES2015 than AMD.
 
 ##### Transpiling
+
 Transpilers should only be used when they bring clear benefits. An example of this is using [TypeScript](http://www.typescriptlang.org/) or [Babel](https://babeljs.io/) to allow using EcmaScript 6 features. When using transpilers, be mindful of polyfills in the generated code, and ensure that performance is good across all supported browsers.
 
 Using CSS preprocessors is OK, and even encouraged. Maintainability and quality of processed code is still more important, though. Don’t use the most esoteric features, unless it clearly adds value (“because you can” != value).
 
-The recommended CSS preprocessor is [SCSS](http://www.sass-lang.com/guide).
+The recommended CSS preprocessor is [SCSS](http://www.sass-lang.com/guide). Also consider using [PostCSS](https://github.com/postcss/postcss).
 
 ##### Browser support
+
 Any customer projects accessible by end users need to support whatever browsers the customer wants to support. For customer admin tools (ie. only used by their internal staff), we should push for only guaranteeing support for latest versions of Chrome and Firefox. For internal projects, we can assume latest versions of Chrome and Firefox.
 
 When adding support for additional browsers infers very little overhead, we should support as broadly as possible. The two latest versions of all major browsers is the industry standard. Legacy versions of IE must be supported according to requirements from our customers.
 
-##### Testing
-[Karma](http://karma-runner.github.com/)/[Mocha](http://visionmedia.github.com/mocha/)/[Chai](http://chaijs.com/)/[Sinon](http://sinonjs.org/) seems to be the best combo ATM but since its an evolving technology it may change in the very near future.
-
 ### Testing
-All projects should have unit tests. Unit tests should cover as much of the code base as is practically possible. The recommended minimum level of code coverage is 70%.
-
-It is recommended that all projects have functional tests covering the main usage of the service.
 
-We also encourage higher level testing, outlined in [Software Testing Practices](https://docs.google.com/a/aptoma.com/document/d/1Sy6lmekbB7ZH-upI99HhJzPJNDXtBrz5cnSf3f0hTKs/edit#).
-
-A few guidelines for unit tests:
-
-- For PHP, PHPUnit is the recommended assertion library and test runner.
-- The test suite should be easy to run from the command line and from a CI server.
-- Tests should generally only test public methods.
-- Tests should be able to run without a network connection, any external dependencies should be mocked.
+All projects should have tests. Your test coverage should give all contributing developers high confidence of the correctness of the code.
```

Comment:
Thumbs up!

How would one verify that "contributing developers high confidence of the correctness of the code", in practice?